src/world_interface/simulator_interface.cpp

```diff
@@ -106,12 +108,21 @@ static void openCamera(CameraID cam, std::optional<std::vector<double>> list1d=s
 void initCameras() {
 	auto cfg = cam::readConfigFromFile(Constants::AR_CAMERA_CONFIG_PATH);
 	cameraConfigMap[Constants::AR_CAMERA_ID] = cfg;
-	//openCamera(Constants::AR_CAMERA_ID);
+	// openCamera(Constants::AR_CAMERA_ID);
 	openCamera("front", cfg.intrinsicParams->getIntrinsicList());
 	openCamera("rear", cfg.intrinsicParams->getIntrinsicList());
 	openCamera("upperArm", cfg.intrinsicParams->getIntrinsicList());
 }
 
+void initMotors() {
+	// initializes map of motor ids and shared ptrs of their objects
+	for (auto const& x : motorNameMap) {
```

Comment:
We put `const` before type, so can you change this to `const auto& x`? We should add a rule to the clang-format for that later.